src/interface/src/app/plan/create-scenarios/create-scenarios.component.html

```diff
@@ -6,12 +6,17 @@
           <input
             formControlName="scenarioName"
             matInput
-            placeholder="Scenario Name" />
+            placeholder="Scenario Name:" />
         </mat-form-field>
       </form>
+      <div *ngIf="formGroups[0].invalid && formGroups[0].touched" class="error">
```

Comment:
I'm mostly just curious, but is there a way to address this by name instead of by array index? It feels a little fragile to use an index.